### PR TITLE
Update naming

### PR DESCRIPTION
**Pull request by annacuomo**
* Update naming
* change to only run plink jobs on relevant genes for cell type at hand (check syntax?)

### CODE REVIEW

File: batch.py [hunk 1]
```diff
@@ -694,6 +694,20 @@ def crm_pipeline(
...
+        expression_tsv_path = dataset_path(
+            os.path.join(
+                expression_files_prefix,
+                'expression_files',
+                f'{celltype}_expression.tsv',
+            )
+        )
+        plink_genes.extend(extract_genes(genes_of_interest, expression_tsv_path))
+    plink_genes = list(sorted(set(plink_genes)))
+    print(f'Genes: {plink_genes}')
```
Comment (illusional) [inline batch.py]: Just generally speaking, does this print a lot of stuff? Print statement are good for debugging, unless they clog up the logs.

File: batch.py [hunk 2]
```diff
@@ -694,6 +694,20 @@ def crm_pipeline(
     celltype_list = celltypes.split(' ')
     logging.info(f'Cell types to run: {celltype_list}')
 
+    # only run this for relevant genes
+    plink_genes: List[str] = []
```
Comment (illusional) [inline batch.py]: You could make this a set to start, and then you don't need to extend + set at the end. In the end, not a big deal but would be a little cleaner, eg:

```python
_plink_genes = set()
for celltype in celltype_list:
    # ...
    _plink_genes |= set(extract_genes(...))

plink_genes = list(sorted(_plink_genes))
```

File: batch.py [hunk 3]
```diff
@@ -774,17 +788,17 @@ def manage_concurrency_for_job(job: hb.job.Job):
 
             plink_output_prefix = gene_dict[gene]['plink']
             # prepare input files
-            prepare_input_job = batch.new_python_job(f'Prepare inputs for: {gene}')
-            manage_concurrency_for_job(prepare_input_job)
-            copy_common_env(prepare_input_job)
+            run_job = batch.new_python_job(f'Run association for: {gene}, {celltype}')
```
Comment (illusional) [inline batch.py]: Not a big deal, but if celltype is "more important", consider putting that first?

File: batch.py [hunk 4]
```diff
@@ -774,17 +788,17 @@ def manage_concurrency_for_job(job: hb.job.Job):
...
+            manage_concurrency_for_job(run_job)
+            copy_common_env(run_job)
             dependency = dependencies_dict.get(gene)
             if dependency:
-                prepare_input_job.depends_on(dependency)
-            prepare_input_job.image(CELLREGMAP_IMAGE)
+                run_job.depends_on(dependency)
```
Comment (illusional) [inline batch.py]: Can use the walrus operator to clean this up slightly.
```python
if dependency := dependencies.get(gene):
    run_job.depends_on(dependency)
```

**Review by illusional — APPROVED** [review]
LGTM, mostly just small non-required suggestions

File: batch.py (hunk 2 above)
Comment (lgruen) [reply batch.py]: Also keeps memory usage lower throughout.

**Review by MattWellie — APPROVED** [review]
LGTM

**Pull request merged**